Gemfile

```diff
@@ -51,8 +51,8 @@ group :development, :staging, :test do
   gem 'sqlite3'
 end
 
-group :staging, :production do
-  gem 'mysql', "~> 2.9.0"
+group :production, :staging do
+  gem 'mysql2', '0.3.10'
```

Comment:
Do we actually need to pin to this version, or can we float against any 0.3 release?